refarch-gateway/README.md

```diff
@@ -78,3 +79,28 @@ spring:
             # needed for userInfo endpoint
             scope: profile, openid
 ```
+
+### Hazelcast
+
+Beside the already mentioned properties Hazelcast also has the following requirements.
+
+#### Modular java
+See https://docs.hazelcast.com/hazelcast/5.5/getting-started/install-hazelcast#using-modular-java
+
+Following Java options need to be set.
+For the gateway image this can be done with `JAVA_OPTS_APPEND`.
+```
+--add-modules java.se \
+  --add-exports java.base/jdk.internal.ref=ALL-UNNAMED \
+  --add-opens java.base/java.lang=ALL-UNNAMED \
+  --add-opens java.base/java.nio=ALL-UNNAMED \
+  --add-opens java.base/sun.nio.ch=ALL-UNNAMED \
+  --add-opens java.management/sun.management=ALL-UNNAMED \
+  --add-opens jdk.management/com.ibm.lang.management.internal=ALL-UNNAMED \
+  --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED
```

Comment:
Is setting this relevant for the `hazelcast-local` profile as well? Besides, I don't really understand the benefits of the `hazelcast-local` profile. Why should I need Hazelcast locally when I only run instance of the gateway?

Reply:
Theoretically yes but as always only one instance running not really required in my opinion.